### PR TITLE
Add Bender Dependencies (1)

### PR DESCRIPTION
**Pull request by WRoenninger**
First part of #388. This adds the soucre dependencies in a seperate folder `deps/` generated by bender in Part 2.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -31,3 +31,7 @@ stdout/
 work-dpi/
 tb/riscv-isa-sim/
 work-*/*
+deps/fpu/src/common_cells*
+deps/fpu/src/fpu_div_sqrt_mvp*
+deps/fpu/tb*
```
Comment (zarubaf) [inline .gitignore]: Why do we ignore those files?

Comment (WRoenninger) [reply .gitignore]: These folders are git submodules inside `fpnew`, which get automatically checked out with bender.

Comment (WRoenninger) [reply .gitignore]: `common_cells` and `fpu_div_sqrt_mvp` are checked out separately in `deps/`

Comment (zarubaf) [reply .gitignore]: Is there a reason to:
1. Don't check them in?
2. I think you can disable the recursive checkout in the `Bender.yml`, please help @fabianschuiki

Comment (WRoenninger) [reply .gitignore:36]: Recursive checkout disable is currently not supported by bender.

Comment (WRoenninger) [reply .gitignore:36]: The checked out submodules get checked out by bender anyways, except the test bench of `fpnew`, which  does not matter in the context of Ariane.

**Pull request merged**